client/src/components/MapWithMarker.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+
+const MapWithAMarker = withGoogleMap(props => (
```

Comment:
The file is called `MapWithMarker` so this component should be named the same.

Reply:
We have renamed it to the name of the file.